### ISSUE TITLE
v1.7.0 crashes on startup

### ISSUE THREAD

**Issue by hotburger** [FrostbyteGames1/Frostbytes-Slabs-and-Stairs#10]
Happens on both 1.20.1 and 1.20.2.

To reproduce:
1. Install fabric api and this mod
2. Attempt to launch the game

```
Caused by: java.lang.IllegalArgumentException: input block must have the 'axis' property
	at net.fabricmc.fabric.api.registry.StrippableBlockRegistry.requireNonNullAndAxisProperty(StrippableBlockRegistry.java:64)
	at net.fabricmc.fabric.api.registry.StrippableBlockRegistry.register(StrippableBlockRegistry.java:50)
	at net.frostbyte.slabsandstairs.SlabsAndStairs.onInitialize(SlabsAndStairs.java:28)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47)
	... 10 more
```

https://mclo.gs/0DEXfNv

**Comment by hotburger** [comment]
Unrelated, but thank you for making this mod. To me this is the only sane stairs and slabs mod that fits in with the vanilla game.

**Comment by FrostbyteGames1** [comment]
> Unrelated, but thank you for making this mod. To me this is the only sane stairs and slabs mod that fits in with the vanilla game.

Thank you for the kind words! That was actually the reason I decided to make this mod in the first place, I felt like the existing implementations either did too much or not enough.

**Comment by FrostbyteGames1** [comment]
Fixed in v1.7.1
